src/OutputFormatter.php

```diff
@@ -143,62 +143,62 @@ public function spaceBeforeOpeningBrace(): string
     /**
      * Runs the given code, either swallowing or passing exceptions, depending on the `ignoreExceptions` setting.
      */
-    public function safely(callable $cCode): ?string
+    public function safely(callable $callable): ?string
     {
         if ($this->outputFormat->getIgnoreExceptions()) {
             // If output exceptions are ignored, run the code with exception guards
             try {
-                return $cCode();
+                return $callable();
             } catch (OutputException $e) {
                 return null;
             } // Do nothing
         } else {
             // Run the code as-is
-            return $cCode();
+            return $callable();
         }
     }
 
     /**
      * Clone of the `implode` function, but calls `render` with the current output format instead of `__toString()`.
      *
-     * @param array<array-key, Renderable|string> $aValues
+     * @param array<array-key, Renderable|string> $values
      */
-    public function implode(string $sSeparator, array $aValues, bool $bIncreaseLevel = false): string
+    public function implode(string $separator, array $values, bool $increaseLevel = false): string
     {
         $result = '';
         $outputFormat = $this->outputFormat;
-        if ($bIncreaseLevel) {
+        if ($increaseLevel) {
             $outputFormat = $outputFormat->nextLevel();
         }
-        $bIsFirst = true;
-        foreach ($aValues as $mValue) {
-            if ($bIsFirst) {
-                $bIsFirst = false;
+        $isFirst = true;
+        foreach ($values as $value) {
+            if ($isFirst) {
+                $isFirst = false;
             } else {
-                $result .= $sSeparator;
+                $result .= $separator;
             }
-            if ($mValue instanceof Renderable) {
-                $result .= $mValue->render($outputFormat);
+            if ($value instanceof Renderable) {
+                $result .= $value->render($outputFormat);
             } else {
-                $result .= $mValue;
+                $result .= $value;
             }
         }
         return $result;
     }
 
-    public function removeLastSemicolon(string $sString): string
+    public function removeLastSemicolon(string $string): string
     {
         if ($this->outputFormat->getSemicolonAfterLastRule()) {
-            return $sString;
+            return $string;
         }
-        $sString = \explode(';', $sString);
-        if (\count($sString) < 2) {
-            return $sString[0];
+        $parts = \explode(';', $string);
```

Comment:
I'd also noted the rewriting of the parameter and thought a fresh local variable would be better 👍